### ISSUE TITLE
Activity log all response properties empty result

### ISSUE THREAD

**Issue by mvput** [Azure/azure-libraries-for-net#178]
When using the `WithAllPropertiesInResponse` on the `ActivityLogs.DefineQuery()` it results in empty responses. 

After some debugging the problem that the following line `string.Join(",", this.responsePropertySelector.OrderBy(o => o))` results in a empty string instead of a null string. After that, the following part is called. Because the string is empty it will append a `$select=` query.

```
if (select != null)
{
     _queryParameters.Add(string.Format("$select={0}", System.Uri.EscapeDataString(select)));
}
```

**Comment by hovsepm** [comment]
@mvput Thanks for reporting this. I'll make sure it is fixed in the nearest release.

**Comment by hovsepm** [comment]
Fixed. Will be available in v1.7
